### ISSUE TITLE
kaggle api in GCP

### ISSUE THREAD

**Issue by aaryapatel007** [Kaggle/kaggle-api#100]
how to use kaggle api to download the dataset on GCP bucket?

**Comment by rysteboe** [comment]
I don't understand what you mean.  To download a dataset on Kaggle through the API, follow the setup instructions and run `kaggle datasets download $DATASET`.

**Comment by rysteboe** [comment]
Or do you mean downloading a dataset TO a GCP bucket?  If so, then that is not supported.

**Comment by aaryapatel007** [comment]
![2018-09-29](https://user-images.githubusercontent.com/30631425/46241328-e6f37c80-c3d4-11e8-92c7-459c7251acba.png)
How to fix this?

**Comment by rysteboe** [comment]
Sorry for the delayed response here.  To the best of my knowledge, we've never done any testing with Kaggle inside of Datalab, which means that I can't really offer you much support here.  I'm sorry, I know that's not particularly helpful.  I'll keep this on our radar, but I don't have any ETA on it.

**Comment by zfanswer** [comment]
same issue +1
![image](https://user-images.githubusercontent.com/1787565/49714946-31755d00-fc92-11e8-948c-e0d4a8e131e2.png)

**Comment by AlanRMalta** [comment]
Likewise (but from AWS).

**Comment by hokowi** [comment]
I found a work-around. I manually uploaded the kaggle.json file from my local drive (to any folder via datalab console) then move it into the "/content/.kaggle" folder through command line - then once I downloaded the competition data, I "mv" the file out from ".kaggle" folder to the destination. 
![screenshot 2019-02-10 at 11 23 46 pm](https://user-images.githubusercontent.com/22044450/52535607-70f02080-2d8b-11e9-9c4f-318804086400.png)

**Comment by ram4189** [comment]
Even though I installed kaggle through both pip3 and pip still it does not consider kaggle as command.
However I saw it in /.local/bin/ . Also there is problem with downloading the competition dataset microsoft malware challenge. However with -f option I was able to download all the files one by one. 

~/.local/bin/kaggle competitions download malware-classification      <gives error 404 not found>
However below works
------------------------
~/.local/bin/kaggle competitions download malware-classification -f file1
~/.local/bin/kaggle competitions download malware-classification -f file2
~/.local/bin/kaggle competitions download malware-classification -f file3

Credit: https://github.com/Kaggle/kaggle-api
